site/content/en/guides/consumer/function/catalog/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Function Catalog"
+linkTitle: "Function Catalog"
+type: docs
```

Comment:
Add weight field so this page appears above exporting workflows page

Reply:
Done.